src/app-layer-htp.h

```diff
@@ -158,6 +166,12 @@ typedef struct HTPCfgRec_ {
     int                 randomize;
     int                 randomize_range;
     int                 http_body_inline;
+
+    /* flash decompression */
+    int                 decomp_swf_enabled;
+    int                 decomp_swf_type;
```

Comment:
instead of int, here we can use the enum defined above. Makes things more clear.